spec/Section 3 -- Type System.md

```diff
@@ -856,7 +856,8 @@ Produces the ordered result:
 **Result Coercion**
 
 Determining the result of coercing an object is the heart of the GraphQL
-executor, so this is covered in that section of the spec.
+executor, so this is covered in the sub-section 
+[6.4.3 Value Completion](#sec-Value-Completion) in Execution section.
```

Comment:
I am not sure about pointing to the section numbers like that. 

I think referring to it more like:

```suggestion
executor, so this is covered in the sub-section [Value Completion](#sec-Value-Completion) in Execution section.
```

But the whole sentence now does not read so well anymore :)